### PR TITLE
Jd/lits structs

### PR DESCRIPTION
**Pull request by jd-lara**
@rodrigomha, this PR moves the LITS structs to PowerSystems. DynLines is not here since it has the same template as a regular line.

### CODE REVIEW

File: src/descriptors/power_system_structs.json [hunk 1]
```diff
@@ -1771,13 +1771,51 @@
...
+        "comment": "power system internal reference, do not modify",
+        "data_type": "InfrastructureSystemsInternal",
+        "internal_default": "InfrastructureSystemsInternal()"
+      }
+    ],
+    "supertype": "AVR"
+  },
+  {
+    "struct_name": "AVRSimple",
+    "docstring": "Parameters of a simple proportional AVR in the derivative of EMF\ni.e. an integrator controller on EMF",
```
Comment (daniel-thom) [inline src/descriptors/power_system_structs.json]: I don't think we want to include formatting characters like \n in these strings.  A higher level could do that automatically.  But, I'm open.

Comment (jd-lara) [reply src/descriptors/power_system_structs.json]: I was getting issues with the JSON format. What do you suggest?

Comment (daniel-thom) [reply src/descriptors/power_system_structs.json]: Don't include \n in the string.  If we want line wrapping in the generated docstrings, we can do later in generate_structs.jl (later).

Comment (jd-lara) [reply src/descriptors/power_system_structs.json]: let's solve this later.

**Pull request merged**